repo/db/db.go

```diff
@@ -246,7 +253,9 @@ func initDatabaseTables(db *sql.DB, pin string) error {
 
     create table cafe_client_messages (id text not null, peerId text not null, clientId text not null, date integer not null, primary key (id, clientId));
     create index cafe_client_message_clientId on cafe_client_messages (clientId);
-    create index cafe_client_message_date on cafe_client_messages (date);
+	create index cafe_client_message_date on cafe_client_messages (date);
+		
+	create table cafe_dev_tokens (id text primary key not null, token text not null, created integer not null);
```

Comment:
can we use `date` instead of `created`? i've been trying to stick to the convention of using date when there is only one, as in, no corresponding `updated` date.

Reply:
sure!